brian_cheung/the_basics.md

```diff
@@ -114,15 +114,14 @@ the index value of Michelangelo would be 2 because an array starts from 0 not 1
 
 **Answer 8A:**
 ```ruby
-# write your code here
-```
 puts jurassic_park{"Triceratops"}
+```
 
 * In Ruby, write code to add "Dr. Satler" to the "Humans" of `jurassic_park`.
 
 **Answer 8B:**
 ```ruby
-# write your code here
+jurassic_park{"Humans"}.push"Dr. Satler"
 ```
 
```

Comment:
It's also not going to work here. Play around in IRB and look up the format - you're close, but you're using the wrong types of punctuation.